src/Raven.Server/Commercial/ApiHttpClient.cs

```diff
@@ -8,7 +8,14 @@ namespace Raven.Server.Commercial
 {
     public static class ApiHttpClient
     {
-        public const string ApiRavenDbNet = "https://api.ravendb.net";
+        public static string ApiRavenDbNet 
+        {
+            get
+            {
+                var envValue = Environment.GetEnvironmentVariable("API_RAVENDB_NET_URL");
```

Comment:
I do not like it. Better to have `RAVEN_API_MODE` with `Stable` and `Test` values and hardcode the URLs.